analysis/lang/cro/analyzer_cro.go

```diff
@@ -0,0 +1,64 @@
+//  Copyright (c) 2018 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cro
+
+import (
+	"github.com/blevesearch/bleve/analysis"
+	"github.com/blevesearch/bleve/registry"
+
+	"github.com/blevesearch/bleve/analysis/token/lowercase"
+	"github.com/blevesearch/bleve/analysis/tokenizer/unicode"
+)
+
+// http://nlp.ffzg.hr/resources/tools/stemmer-for-croatian/
+
+const AnalyzerName = "cro"
```

Comment:
I am not sure whether "cro" is the apt name here. Most of the language analyzers follow a name inspired by the language code (ISO_639-1).
So hr or hrv might be better?

Reply:
I thought that `cro` would be most recognizable one, but `hr` sounds good. Thanks for the tip.